test/server/cards/03_TWI/units/EncouragingLeadership.spec.ts

```diff
@@ -0,0 +1,40 @@
+describe('EncouragingLeadership', function () {
+    integration(function (contextRef) {
+        describe('EncouragingLeadership\'s ability', function () {
+            it('Gives +1/+1 for this phase to allies', function () {
```

Comment:
nit: if there's just one `it` in the `describe` you can just remove the `describe` part and put that text inside of the `it` name (e.g., `it('Encouraging Leadership\'s ability gives +1/+1 for this phase to allies')`)